config/locales/views/grade_entry_forms/en.yml

```diff
@@ -2,6 +2,8 @@
 en:
   grade_entry_forms:
     add_column: Add a column
+    successfully_deleted: Marks Spreadsheet was successfully deleted.
+    delete: Delete
```

Comment:
There should already be text defined for this under `common`, make sure to use that